posts/inside-rust/2020-11-12-source-based-code-coverage.md

```diff
@@ -60,16 +60,18 @@ Work on the implementation [began back in April][MCP], and [many PRs
 later][PRs], it’s ready for you to try. All you need is a recent nightly and
 a tool to read the coverage reports.
 
-[Take a look at this guide to get started][guide]. If you spot any issues,
-please [report them]. It’s a huge help!
+The easiest way to get stared with source-based coverage is through 
+[`cargo-llvm-cov`](cargo-llvm-cov).
```

Comment:
Square brackets should be used around local link references (or at least in markdown it's needed, I could be wrong about here :) )

```suggestion
[`cargo-llvm-cov`][cargo-llvm-cov].
```